src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/TextareaChoiceListProvider.java

```diff
@@ -154,13 +169,15 @@ public String getDefaultChoice()
      * @param defaultChoice
      * @param addEditedValue
      * @param whenToAdd
+     * @param addToTop
      */
     @DataBoundConstructor
-    public TextareaChoiceListProvider(String choiceListText, String defaultChoice, boolean addEditedValue, WhenToAdd whenToAdd)
+    public TextareaChoiceListProvider(String choiceListText, String defaultChoice, boolean addEditedValue, WhenToAdd whenToAdd, boolean addToTop)

```

Comment:
Would you add a backward compatible constructor?
example: https://github.com/jenkinsci/naginator-plugin/blob/naginator-1.17/src/main/java/com/chikli/hudson/plugin/naginator/NaginatorPublisher.java#L53

Adding  a backward compatible constructor preserves binary compatibilities,
and you no longer need to edit existing test codes.
(I didn't add backward compatible constructor in previous changes for extebsible-choice-parameter, and I know that was a really bad idea...)